dd-sdk-android/src/test/kotlin/com/datadog/android/utils/extension/IntExt.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.utils.extension
+
+import com.datadog.android.log.internal.domain.LogGenerator
+import com.datadog.android.log.model.LogEvent
+
+fun Int.asLogStatus(): LogEvent.Status {
+    return when (this) {
+        android.util.Log.ASSERT -> LogEvent.Status.CRITICAL
```

Comment:
Why don't you `import android.util.Log`

Reply:
I think I just copy pasted this code from somewhere else without paying attention to that. Will change it now :)